borg/_hashindex.c

```diff
@@ -77,6 +79,7 @@ static int hash_sizes[] = {
 
 #define HASH_MIN_LOAD .25
 #define HASH_MAX_LOAD .75  /* don't go higher than 0.75, otherwise performance severely suffers! */
+#define HASH_MAX_EFF_LOAD .85
```

Comment:
hmm, i read the 0.83 value from a table (iirc from same guy).
if we could even go to 0.93, that would save us some resizes.